ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gc2VhcmNoLXJlc3VsdHMiLCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjgzNTk3NzUzMi4xNzQ1NDQ4MTI5In0sImFub25JZCI6ImU0MzJkNGYyZjRkYTJmNDUifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnouaXQvcGFzc2VuZ2VyY2Fycy9idXkvbmV3LWNhci9zZWFyY2gtcmVzdWx0cy5odG1sL3ZlaGljbGVDYXRlZ29yeS1uZXctcGFzc2VuZ2VyLWNhcnMvYnJhbmQtTWVyY2VkZXMtQU1HL21vZGVsSWRlbnRpZmllci1TTF9DTEFTUy8%2FZW1oc29ydFR5cGU9cHJpY2UtYXNjJmVtaGJvZHlUeXBlPVJPQURTVEVSIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI2MyIsImNvbnRlbnRab25lcyI6W119LCJjb25zZW50cyI6W10sImFjY291bnQiOnt9LCJfdG9vbHNFdmVudExpbmtJZCI6IjQ0OTI2MjEzNDc1Njk5NzUiLCJkZWJ1ZyI6eyJleHBsYW5hdGlvbnMiOnRydWV9fQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x5607995a9cea <unknown>
#1 0x56079905a5f0 <unknown>
#2 0x560799040e86 <unknown>
#3 0x56079903f79a <unknown>
#4 0x56079903feff <unknown>
#5 0x56079903fe54 <unknown>
#6 0x56079905e7ff <unknown>
#7 0x5607991039aa <unknown>
#8 0x5607990d1662 <unknown>
#9 0x5607990f7660 <unknown>
#10 0x5607990d1433 <unknown>
#11 0x56079909dea3 <unknown>
#12 0x56079909eb01 <unknown>
#13 0x56079956eb3b <unknown>
#14 0x560799572a21 <unknown>
#15 0x560799555c32 <unknown>
#16 0x560799573594 <unknown>
#17 0x560799539eef <unknown>
#18 0x560799597d98 <unknown>
#19 0x560799597f76 <unknown>
#20 0x5607995a8b36 <unknown>
#21 0x7fd9c2e9caa4 <unknown>
#22 0x7fd9c2f29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJpdGVtQWN0aW9uIjpudWxsLCJzb3VyY2UiOnsicGFnZVR5cGUiOiJkZWZhdWx0IiwibG9jYWxlIjoiaXRfSVQiLCJjb250ZW50Wm9uZXMiOltdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvYnV5L25ldy1jYXIvc2VhcmNoLXJlc3VsdHMuaHRtbC92ZWhpY2xlQ2F0ZWdvcnktbmV3LXBhc3Nlbmdlci1jYXJzL2JyYW5kLU1lcmNlZGVzLUFNRy9tb2RlbElkZW50aWZpZXItU0xfQ0xBU1MvP2VtaHNvcnRUeXBlPXByaWNlLWFzYyZlbWhib2R5VHlwZT1ST0FEU1RFUiIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNjMifSwiZmxhZ3MiOnsicGFnZVZpZXciOnRydWV9LCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjgzNTk3NzUzMi4xNzQ1NDQ4MTI5In0sImFub25JZCI6ImU0MzJkNGYyZjRkYTJmNDUifSwicGVyZm9ybWFuY2UiOnsic2RrUGFyc2VUaW1lIjoxfSwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfSwiY2F0YWxvZyI6e30sImNvbnNlbnRzIjpbXSwiYWNjb3VudCI6e30sIl90b29sc0V2ZW50TGlua0lkIjoiNzc2MDA4MzY1NjU3NDA0MiJ9: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x5607995a9cea <unknown>
#1 0x56079905a5f0 <unknown>
#2 0x560799040e86 <unknown>
#3 0x56079903f79a <unknown>
#4 0x56079903feff <unknown>
#5 0x56079903fe54 <unknown>
#6 0x56079905e7ff <unknown>
#7 0x5607991039aa <unknown>
#8 0x5607990d1662 <unknown>
#9 0x5607990f7660 <unknown>
#10 0x5607990d1433 <unknown>
#11 0x56079909dea3 <unknown>
#12 0x56079909eb01 <unknown>
#13 0x56079956eb3b <unknown>
#14 0x560799572a21 <unknown>
#15 0x560799555c32 <unknown>
#16 0x560799573594 <unknown>
#17 0x560799539eef <unknown>
#18 0x560799597d98 <unknown>
#19 0x560799597f76 <unknown>
#20 0x5607995a8b36 <unknown>
#21 0x7fd9c2e9caa4 <unknown>
#22 0x7fd9c2f29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJTZWFyY2ggUmVzdWx0IFBhZ2UgLSBDYXJzIiwidXNlciI6eyJhdHRyaWJ1dGVzIjp7ImdhQ2xpZW50SUQiOiI4MzU5Nzc1MzIuMTc0NTQ0ODEyOSJ9LCJhbm9uSWQiOiJlNDMyZDRmMmY0ZGEyZjQ1In0sInNvdXJjZSI6eyJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvYnV5L25ldy1jYXIvc2VhcmNoLXJlc3VsdHMuaHRtbC92ZWhpY2xlQ2F0ZWdvcnktbmV3LXBhc3Nlbmdlci1jYXJzL2JyYW5kLU1lcmNlZGVzLUFNRy9tb2RlbElkZW50aWZpZXItU0xfQ0xBU1M%2FZW1oc29ydFR5cGU9cHJpY2UtYXNjJmVtaGJvZHlUeXBlPVJPQURTVEVSIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI2MyIsImNvbnRlbnRab25lcyI6W119LCJjb25zZW50cyI6W10sImFjY291bnQiOnt9LCJfdG9vbHNFdmVudExpbmtJZCI6IjQ5MDA0MzY1NjIxNzM3NjgiLCJkZWJ1ZyI6eyJleHBsYW5hdGlvbnMiOnRydWV9fQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x5607995a9cea <unknown>
#1 0x56079905a5f0 <unknown>
#2 0x560799040e86 <unknown>
#3 0x56079903f79a <unknown>
#4 0x56079903feff <unknown>
#5 0x56079903fe54 <unknown>
#6 0x56079905e7ff <unknown>
#7 0x5607991039aa <unknown>
#8 0x5607990d1662 <unknown>
#9 0x5607990f7660 <unknown>
#10 0x5607990d1433 <unknown>
#11 0x56079909dea3 <unknown>
#12 0x56079909eb01 <unknown>
#13 0x56079956eb3b <unknown>
#14 0x560799572a21 <unknown>
#15 0x560799555c32 <unknown>
#16 0x560799573594 <unknown>
#17 0x560799539eef <unknown>
#18 0x560799597d98 <unknown>
#19 0x560799597f76 <unknown>
#20 0x5607995a8b36 <unknown>
#21 0x7fd9c2e9caa4 <unknown>
#22 0x7fd9c2f29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJpdGVtQWN0aW9uIjpudWxsLCJzb3VyY2UiOnsicGFnZVR5cGUiOiJzZWFyY2hfcmVzdWx0IiwibG9jYWxlIjoiaXRfSVQiLCJjb250ZW50Wm9uZXMiOltdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvYnV5L25ldy1jYXIvc2VhcmNoLXJlc3VsdHMuaHRtbC92ZWhpY2xlQ2F0ZWdvcnktbmV3LXBhc3Nlbmdlci1jYXJzL2JyYW5kLU1lcmNlZGVzLUFNRy9tb2RlbElkZW50aWZpZXItU0xfQ0xBU1M%2FZW1oc29ydFR5cGU9cHJpY2UtYXNjJmVtaGJvZHlUeXBlPVJPQURTVEVSIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI2MyJ9LCJmbGFncyI6eyJwYWdlVmlldyI6dHJ1ZX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiODM1OTc3NTMyLjE3NDU0NDgxMjkifSwiYW5vbklkIjoiZTQzMmQ0ZjJmNGRhMmY0NSJ9LCJwZXJmb3JtYW5jZSI6e30sImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX0sImNhdGFsb2ciOnt9LCJjb25zZW50cyI6W10sImFjY291bnQiOnt9LCJfdG9vbHNFdmVudExpbmtJZCI6IjM1NDIwNzUzNjIxMDg1OSJ9: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x5607995a9cea <unknown>
#1 0x56079905a5f0 <unknown>
#2 0x560799040e86 <unknown>
#3 0x56079903f79a <unknown>
#4 0x56079903feff <unknown>
#5 0x56079903fe54 <unknown>
#6 0x56079905e7ff <unknown>
#7 0x5607991039aa <unknown>
#8 0x5607990d1662 <unknown>
#9 0x5607990f7660 <unknown>
#10 0x5607990d1433 <unknown>
#11 0x56079909dea3 <unknown>
#12 0x56079909eb01 <unknown>
#13 0x56079956eb3b <unknown>
#14 0x560799572a21 <unknown>
#15 0x560799555c32 <unknown>
#16 0x560799573594 <unknown>
#17 0x560799539eef <unknown>
#18 0x560799597d98 <unknown>
#19 0x560799597f76 <unknown>
#20 0x5607995a8b36 <unknown>
#21 0x7fd9c2e9caa4 <unknown>
#22 0x7fd9c2f29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJzZXRfdXNlcl9hdHRyaWJ1dGUiLCJhY2NvdW50Ijp7ImF0dHJpYnV0ZXMiOnsibGFzdFNSUE1vZGVsRmlsdGVyIjoie1wiZGNwX2xhc3RfbW9kZWxmaWx0ZXJcIjpcInNsLWNsYXNzXCIsXCJkY3BfbGFzdF9ib2R5dHlwZWZpbHRlclwiOlwicm9hZHN0ZXJcIixcImRjcF9sYXN0X3NlYXJjaF91cmxcIjpcImh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnouaXQvcGFzc2VuZ2VyY2Fycy9idXkvbmV3LWNhci9zZWFyY2gtcmVzdWx0cy5odG1sL3ZlaGljbGVDYXRlZ29yeS1uZXctcGFzc2VuZ2VyLWNhcnMvYnJhbmQtTWVyY2VkZXMtQU1HL21vZGVsSWRlbnRpZmllci1TTF9DTEFTUz9lbWhzb3J0VHlwZT1wcmljZS1hc2MmZW1oYm9keVR5cGU9Uk9BRFNURVJcIixcInRpbWVzdGFtcFwiOjE3NDU0NDgxMzQ0NTQsXCJleHBpcmF0aW9uXCI6MTIwfSJ9fSwidXNlciI6eyJhdHRyaWJ1dGVzIjp7ImdhQ2xpZW50SUQiOiI4MzU5Nzc1MzIuMTc0NTQ0ODEyOSJ9LCJhbm9uSWQiOiJlNDMyZDRmMmY0ZGEyZjQ1In0sInNvdXJjZSI6eyJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvYnV5L25ldy1jYXIvc2VhcmNoLXJlc3VsdHMuaHRtbC92ZWhpY2xlQ2F0ZWdvcnktbmV3LXBhc3Nlbmdlci1jYXJzL2JyYW5kLU1lcmNlZGVzLUFNRy9tb2RlbElkZW50aWZpZXItU0xfQ0xBU1M%2FZW1oc29ydFR5cGU9cHJpY2UtYXNjJmVtaGJvZHlUeXBlPVJPQURTVEVSIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI2MyIsImNvbnRlbnRab25lcyI6W119LCJjb25zZW50cyI6W10sIl90b29sc0V2ZW50TGlua0lkIjoiMDkzNDg4NDU0MTE3MDY3ODUiLCJkZWJ1ZyI6eyJleHBsYW5hdGlvbnMiOnRydWV9fQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x5607995a9cea <unknown>
#1 0x56079905a5f0 <unknown>
#2 0x560799040e86 <unknown>
#3 0x56079903f79a <unknown>
#4 0x56079903feff <unknown>
#5 0x56079903fe54 <unknown>
#6 0x56079905e7ff <unknown>
#7 0x5607991039aa <unknown>
#8 0x5607990d1662 <unknown>
#9 0x5607990f7660 <unknown>
#10 0x5607990d1433 <unknown>
#11 0x56079909dea3 <unknown>
#12 0x56079909eb01 <unknown>
#13 0x56079956eb3b <unknown>
#14 0x560799572a21 <unknown>
#15 0x560799555c32 <unknown>
#16 0x560799573594 <unknown>
#17 0x560799539eef <unknown>
#18 0x560799597d98 <unknown>
#19 0x560799597f76 <unknown>
#20 0x5607995a8b36 <unknown>
#21 0x7fd9c2e9caa4 <unknown>
#22 0x7fd9c2f29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gcHJvZHVjdCIsInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiODM1OTc3NTMyLjE3NDU0NDgxMjkifSwiYW5vbklkIjoiZTQzMmQ0ZjJmNGRhMmY0NSJ9LCJzb3VyY2UiOnsidXJsIjoiaHR0cHM6Ly93d3cubWVyY2VkZXMtYmVuei5pdC9wYXNzZW5nZXJjYXJzL2J1eS9uZXctY2FyL3Byb2R1Y3QuaHRtbC9NZXJjZWRlcy1BTUctU0wtNDMtUFJFTUlVTS1QTFVTXzA0NTQzMjEzMTNfSVRfMzU0YWUyZjUiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjYzIiwiY29udGVudFpvbmVzIjpbXX0sImNvbnNlbnRzIjpbXSwiYWNjb3VudCI6e30sIl90b29sc0V2ZW50TGlua0lkIjoiMDkyNTQ4MjcxNzU3MDUyOTQiLCJkZWJ1ZyI6eyJleHBsYW5hdGlvbnMiOnRydWV9fQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x5607995a9cea <unknown>
#1 0x56079905a5f0 <unknown>
#2 0x560799040e86 <unknown>
#3 0x56079903f79a <unknown>
#4 0x56079903feff <unknown>
#5 0x56079903fe54 <unknown>
#6 0x56079905e7ff <unknown>
#7 0x5607991039aa <unknown>
#8 0x5607990d1662 <unknown>
#9 0x5607990f7660 <unknown>
#10 0x5607990d1433 <unknown>
#11 0x56079909dea3 <unknown>
#12 0x56079909eb01 <unknown>
#13 0x56079956eb3b <unknown>
#14 0x560799572a21 <unknown>
#15 0x560799555c32 <unknown>
#16 0x560799573594 <unknown>
#17 0x560799539eef <unknown>
#18 0x560799597d98 <unknown>
#19 0x560799597f76 <unknown>
#20 0x5607995a8b36 <unknown>
#21 0x7fd9c2e9caa4 <unknown>
#22 0x7fd9c2f29c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:18 ❌ Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-App-Allure/QA-App-Allure/Tests/Last Seen PDP_attempt_1.png'